projects/storefrontlib/src/shared/services/trigger/config/default-trigger-config.ts

```diff
@@ -0,0 +1,15 @@
+import { TriggerConfig } from './trigger-config';
+import { OutletPosition } from '../../../../cms-structure/outlet/index';
+
+export const DEFAULT_TRIGGER_CONFIG: TriggerConfig = {
+  trigger: {
+    asm: {
```

Comment:
We might end up in the future with different types. To that reason it might be better to wrap the outlet and position into an outlet object:
```typescript
asm: {
  outlet: {
    name: 'cx-storefront'
    position: OutletPosition.BEFORE,
  }
}
```

Reply:
Just looked at the model. I like it, forget ^^.